openpgp/keys_test.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 var hashes = []crypto.Hash{
-	crypto.SHA1,
+	crypto.SHA3_256,
+	crypto.SHA3_512,
```

Comment:
Please leave SHA1 here as well for now, and just add SHA3 at the end of the list in this PR